test/deercreeklabs/node_test_runner.cljs

```diff
@@ -6,7 +6,7 @@
 
 (nodejs/enable-util-print!)
 
-(defn -main [& args]
+(defn -main [& _args]
```

Comment:
Unused binding. This is a style thing. If you lead with an underscore it considers it proper.